### PR TITLE
add link to provider array deprecation error message

### PR DESCRIPTION
**Pull request by carmenberndt**
related: https://github.com/prisma/pris.ly/pull/11

### CODE REVIEW

**Review by tomhoule — APPROVED** [review]

**Comment by pimeys** [comment]
https://pris.ly/prd/multi-provider-deprecation is 404, is this ok still?

**Comment by carmenberndt** [comment]
> https://pris.ly/prd/multi-provider-deprecation is 404, is this ok still?

yes, that's expected, it's defined already before the actual page exists (will be added till the release though: https://prisma-company.slack.com/archives/C5Z9TH6N9/p1603121807040200?thread_ts=1603112542.035700&cid=C5Z9TH6N9)

**Review by pimeys — APPROVED** [review]

**Review by janpio — CHANGES_REQUESTED** [review]
On hold until the link format is clarified: https://github.com/prisma/pris.ly/pull/11#discussion_r509121073

File: libs/datamodel/core/src/diagnostics/warning.rs [hunk 1]
```diff
@@ -13,7 +13,7 @@ pub enum DatamodelWarning {
   #[error("Preview feature \"{}\" is deprecated. The functionality can be used without specifying it as a preview feature.", preview_feature)]
   DeprecatedPreviewFeature { preview_feature: String, span: Span },
 
-  #[error("Using multiple providers is now deprecated. You should use a single provider instead.")]
+  #[error("Using multiple providers is now deprecated. You should use a single provider instead. Read more at https://pris.ly/prd/multi-provider-deprecation")]
```
Comment (janpio) [inline libs/datamodel/core/src/diagnostics/warning.rs:16]:
```suggestion
  #[error("Using multiple providers is now deprecated. You should use a single provider instead. Read more at https://pris.ly/multi-provider-deprecation")]
```
https://github.com/prisma/pris.ly/pull/11 was updated

**Pull request merged**